### DISCUSSION TITLE
Pwa push notification with firebase

### DISCUSSION THREAD

**Discussion by dikahastanto** [quasarframework/quasar#10361] [CLI - PWA mode]
How to create real-time push notification quasar pwa with firebase ?
I already made it, but i can't get token for send spesific device.

**Comment by ontwikkelfabriek** [comment]
@dikahastanto 

I would look elsewhere for support. It's not something Quasar specific. I would try the firebase or workbox community. 
Maybe you could find your answers in Danny's Quasar PWA/firebase/notifications/ect course:
https://www.dconnell.co.uk/#/
